### DISCUSSION TITLE
Feedback about Failsafe mode:

### DISCUSSION THREAD

**Discussion by cgessai** [LostRuins/koboldcpp#199] [General]
Working fine on an old laptop under Win 7 though (predictably) very slow.  It's taking about 8 seconds per token but otherwise appears to be working well with [WizardLM 7B Uncensored Q4_0](https://huggingface.co/TheBloke/WizardLM-7B-uncensored-GGML/blob/main/WizardLM-7B-uncensored.ggmlv3.q4_0.bin) in both Instruct and Story mode.  FWIW, the model has been quantized using the newer May 19th format.  Also, using command line args: **--noavx2 --noblas --nommap --stream --launch** and url with ?streamamount=2

**Comment by LostRuins** [comment]
Great glad to know it works! This will allow even those with ancient devices to try it out.
